ubports-qa

```diff
@@ -50,9 +50,14 @@ class WritableRootFS:
         LOG.debug("Attempting to mount rootfs read-write.")
         ensure_root()
         subprocess.run(["mount", "-o", "rw,remount", "/"])
+        LOG.debug("Making sure we have enough free space for archives.")
+        subprocess.run(["mount", "-t", "tmpfs", "tmpfs", "/var/cache/apt/archives"])
 
     @classmethod
     def attempt_unmount(cls):
+        LOG.debug("Unmounting tmpfs")
+        ensure_root()
+        subprocess.run(["umount", "/var/cache/apt/archives"])
```

Comment:
It might be better to use only one `ensure_root()` call in this function, removing the second one under the `try:` statement.

I think it's okay if this command fails, it should not adversely affect the user, so you're correct to put it outside of a `try:` statement and not set `check=True`.